src/io/internal/pipelines/dicom/apply-pstate-to-image.cxx

```diff
@@ -64,10 +64,14 @@
 #include "dcmtk/dcmdata/dcjson.h"     /* JSON format */
```

Comment:
Since `rapidjson` is now used and there are no occurrences of `DcmJsonFormat`, this include is likely not needed.